scripts/portfolioView.js

```diff
@@ -1,79 +1,107 @@
-var portfolioView = {};
+(function(module){
+  var portfolioView = {};//empty object gets functions from below passed in it
 
-portfolioView.populateFilters = function() {
-  $('article').each(function() {
-    if (!$(this).hasClass('template')) {
-      var val = $(this).attr('data-category');
+//added category from ProjectData to OptionTag filter
+  portfolioView.populateFilters = function() {
+    $('article').each(function() {//index 16-26
+      var val = $(this).attr('data-category');//index 16
       var optionTag = '<option value="' + val + '">' + val + '</option>';
       if ($('#category-filter option[value="' + val + '"]').length === 0) {
         $('#category-filter').append(optionTag);
       }
-    }
-  });
-};
+    });
+  };
 
-portfolioView.handleCategoryFilter = function() {
-  $('#category-filter').on('change', function() {
-    if ($(this).val()) {
-      $('article').hide();
-      $('article[data-category="' + $(this).val() + '"]').fadeIn();
-    } else {
-      $('article').fadeIn(); //fadeOut ????
-      $('.template').hide();
-    }
-  });
-};
+//Filters category-filter when user changes category
+  portfolioView.handleCategoryFilter = function() {
+    $('#category-filter').on('change', function() {
+      if ($(this).val()) {
+        $('article').hide();
+        $('article[data-category="' + $(this).val() + '"]').fadeIn();
+      } else {
+        $('article').fadeIn();
+        $('.template').hide();
+      }
+    });
+  };
+
+//Hides all content and then fades in only content that is clicked on
+  portfolioView.handleMainNav = function () {
+    $('.main-nav').on('click', '.tab', function () { //index 41-42
```

Comment:
if you do refactor naming conventions remember to update this instance of .tab to li